composer.json

```diff
@@ -29,6 +29,7 @@
         "illuminate/support": "^8.0",
         "league/flysystem": "^1.1.3",
         "phpdocumentor/reflection-docblock": "^5.2",
+        "spatie/better-types": "dev-main",
```

Comment:
I get that the `dev-main` is needed for this to work now, but would we wait for `v1` to be released or would we use the latest version that is available when this PR is ready? Personally I'd say we wait for the `v1` release of `spatie/better-types`.

Reply:
Yeah I'm going to release v1 of `better-types` in preparation of this release :)

Reply:
Ah, this is great.
And great work here, btw!